2019-11-27-jekyll/README.md

```diff
@@ -0,0 +1,35 @@
+## 2019-11-27-Jekyll
+
+* First install Jekyll by run command:
+
+    `gem install bundler jekyll`
+
+* To create static site run:
+
+    `jekyll new <site-name>`
+
+* To run the site first `cd <site-name>`, then run the command:
+
+    `bundle exec jekyll serve`
+
+* To add a page, simply add a `<file-name>.markdown` file in the `<site-name>` directory.
+
+* To add a collection, add
+
+    `collections:`
+        `team_members:`
+           `output: true`
+            `permalink: /:collection/:name`
```

Comment:
think you've got an extra space in here - would cause an error if you copy-pasted it

Reply:
Okay thanks, let us fix that.